code/style.css

```diff
@@ -134,7 +159,512 @@ button {
   transition: all 0.3s ease;
 }
 
-button:hover {
+/*Buttons when you're asked if you like the song*/
+#no, #yes {
+  background-color:#2FAEAF;
+  color: #fff;
+  border: solid #2FAEAF;
+  border-radius: 20px;
+  padding: 5px 20px;
+  margin-right: 4px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: 'Montserrat';
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+/*button to rerun*/
+#rerun{
+  background-color:#C716B7;
+  color: #fff;
+  border: solid #C716B7;
+  border-radius: 20px;
+  padding: 16px 40px;
+  margin-right: 4px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: 'Montserrat';
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
+
+}
+#rerun:hover{
+  background-color: #fff;
+  color: #C716B7;
+  border: solid #C716B7;
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+.send-btn:hover {
+  background-color: #fff;
+  color: #C716B7;
+  border: solid #C716B7;
+  opacity: 0.9;
+  transition: all 0.2s ease;
+}
+
+/*Start button that says 'Say Hey'*/
+.Welcome{
+  width: 50%;
+  height: 50px;
+  background-color: #C716B7;
+  align-self: center;
+  border: none;
+  border-radius: 20px;
+  margin-top: auto;
+  font-size: 30px;
+  color: white;
+}
+
+/* Dropdown where you select a beat*/
+#select1, #select2, #select3, #select4, #select5{
+  background: #2FAEAF;
+  color: #fff;
+  font-weight: bold;
+  width: 50%;
+  height: 40px;
+  border-radius: 10px;
+  margin: 20px;
+  font-size: 16px;
+  border: none;
+}
+
+/*sound waves at the top of the page*/
+.spinner {
+  margin: 20px auto;
+  width: 50px;
+  height: 40px;
+  text-align: center;
+  font-size: 10px;
+}
+/*anumated spinner at top*/
+.spinner > div {
+  background-color: #fff;
+  height: 100%;
+  width: 6px;
+  display: inline-block;
+  
+  animation: stretch 1.2s infinite ease-in-out;
+}
+
+.spinner .rect2 {
+  animation-delay: -1.1s;
+}
+
+.spinner .rect3 {
+  animation-delay: -1.0s;
+}
+
+.spinner .rect4 {
+  animation-delay: -0.9s;
+}
+
+.spinner .rect5 {
+  animation-delay: -0.8s;
+}
+
```

Comment:
Can we just notice how cool this looks? I know I mentioned it in the HTML but i'll do it again. Looks so stylish, Nice nice nice!